src/lib_ccx/ccx_decoders_isdb.c

```diff
@@ -357,7 +357,7 @@ static struct ISDBText *allocate_text_node(ISDBSubLayout *ls)
 static int reserve_buf(struct ISDBText *text, size_t len)
 {
 	size_t blen;
-	unsigned char *ptr;
+	unsigned char *ptr; 
```

Comment:
All pointers should be initialized to something (or NULL). Really hate uninitialized variables.